infra/eligibility_screener/environments/test/main.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  environment_name = "test"
+}
+
+provider "aws" {
+  region = "us-east-1"
+  profile   = "wic-mt"
+}
+
+terraform {
+  required_version = "1.2.0"
+
+    backend "s3" {
+    bucket         = "wic-mt-tf-state"
+    key            = "terraform/screener.tfstate"
```

Comment:
this will eventually need to include the environment name, but I know this is moving existing terraform over so might be confusing to rename it as part of this PR.